index.js

```diff
@@ -8,6 +8,10 @@ function fastifyEnv (fastify, opts, done) {
     const config = envSchema(opts)
     const confKey = opts.confKey || 'config'
     fastify.decorate(confKey, config)
+
+    fastify.decorate('getEnvs', () => { return config })
+    fastify.decorateRequest('getEnvs', () => { return config })
```

Comment:
I would consider it a breaking change - the `getEnv` decorator is too much generic and it could be already used by user's application.

Reply:
I agree...  
The name can be changed though, this is the first one that came to mind

Reply:
Resolved as in https://github.com/fastify/fastify-multipart/pull/522#discussion_r1591655656